modules/manage/pages/monitor/monitoring-n1ql-query.adoc

```diff
@@ -1292,11 +1297,11 @@ To get a list of all logged completed requests using the Admin REST API:
 curl -u Administrator:pword http://localhost:8093/admin/completed_requests
 ----
 
-To get a list of all logged completed requests using {sqlpp}, including the query plan:
+To get a list of all logged completed requests using {sqlpp}, including only successful requests:
 
 [source,sqlpp]
 ----
-SELECT *, meta().plan FROM system:completed_requests;
```

Comment:
I do not think we should be removing this section

Reply:
Reinstated